src/components/activity/ActivityFormPanel.svelte

```diff
@@ -167,51 +193,104 @@
 
   <svelte:fragment slot="body">
     {#if $selectedActivity}
+      <div class="activity-header">
+        <div class="activity-header-title-placeholder st-typography-medium">
+          {activityName}
+        </div>
+      </div>
       <fieldset>
-        <label for="id">Activity ID</label>
-        <input class="st-input w-100" disabled name="id" value={id} />
-      </fieldset>
+        <details open style:cursor="pointer">
+          <summary>Trace</summary>
```

Comment:
Can we change the name from `Trace` to `Definition` (or something more descriptive)

Reply:
Will use Definition for now, can do more bikeshedding though.